resources/views/oauth2/consent.blade.php

```diff
@@ -1,50 +1,50 @@
-@extends('layout2')
-
-@section('scripts')
-    {!! HTML::style('assets/css/auth/consent.css') !!}
-    {!! HTML::script('assets/js/oauth2/consent.js') !!}
-@append
+@extends('reactapp_layout')
 
 @section('title')
-<title>Welcome to {!! Config::get('app.app_name') !!} - Request for Permission </title>
-@stop
+    <title>Welcome to {!! Config::get('app.app_name') !!} - Request for Permission </title>
+@append
+@section('meta')
+    <meta http-equiv="X-XRDS-Location" content="{!! URL::action("OpenId\DiscoveryController@idp") !!}"/>
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+@append
+@section('css')
+    {!! HTML::style('assets/css/consent.css') !!}
+@append
 
 @section('header_right')
     @if(Auth::check())
         <div class="row">
             <div class="col-md-6 col-md-offset-8">
-                Welcome, <a target="_self" href="{!! URL::action("UserController@getProfile") !!}">{!!Auth::user()->getIdentifier()!!}</a>
+                Welcome, <a target="_self"
+                            href="{!! URL::action("UserController@getProfile") !!}">{!!Auth::user()->getIdentifier()!!}</a>
             </div>
         </div>
     @endif
 @stop
 
-@section('content')
-<div class="container">
-    <div class="row">
-        <div class="col-md-6 col-md-offset-3">
-            <div class="row">
-                <div class="col-md-12 col-xs-12">
```

Comment:
this part is missing
should not be removed 
ant the dev info email should be replaced with contact email if its set on the oauth2 app

Reply:
Done. Now the email is extracted from the oauth2 app contacts and the html section you've mentioned as missing was replaced by the React counterpart